types/office-js-preview/index.d.ts

```diff
@@ -91152,6 +91699,16 @@ declare namespace Word {
          * [Api set: WordApi 1.5]
          */
         getFootnoteBody(): Word.Body;
+        /**
+         * Gets the paragraph by its unique local ID. Throws an `ItemNotFound` error if the collection is empty.
+         *
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param id Required. Unique local ID in GUID format. Note that the ID differs across sessions and coauthors.
```

Comment:
There's more than one GUID format. From wikipedia:

> "The most used format is the 8-4-4-4-12 format, xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx, where every x represents 4 bits. Other well-known formats are the 8-4-4-4-12 format with braces, {xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx}, like in Microsoft's systems, e.g. Windows, or xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx, where all hyphens are removed. "

We need to specify which of these is acceptable. It might be that more than one is.

Reply:
Updated